src/Native/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -377,3 +384,11 @@ Returns true/false based on if existing ssl session was re-used or not.
 Shims the SSL_session_reused macro.
 */
 extern "C" bool SslSessionReused(SSL* ssl);
+
```

Comment:
I see that bool is being used in a signature we already checked in (SslSessionReused). Can you fix that one as well?